addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRuleUnitTest.java

```diff
@@ -372,7 +403,10 @@ private static class ListLinuxDirsOnAttack extends ListDirsOnAttack {
         private static final String DIRS_LISTING =
                 "<td><a href=\"/bin/\">bin</a></td>"
                         + "<td><a href=\"/etc/\">etc</a></td>"
-                        + "<td><a href=\"/boot/\">boot</a></td>";
+                        + "<td><a href=\"/boot/\">boot</a></td>"
+                        + "<td><a href=\"/proc/\">etc</a></td>"
+                        + "<td><a href=\"/tmp/\">etc</a></td>"
```

Comment:
Are these purposefully mismatched? If so please add a comment

Reply:
I'm sorry, this is my typo